electrumsv/gui/qt/request_list.py

```diff
@@ -163,49 +163,46 @@ def on_update(self) -> None:
 
         wallet = self._account._wallet
         rows = wallet.data.read_payment_requests(account_id=self._account_id,
-            flags=PaymentFlag.NONE, mask=PaymentFlag.ARCHIVED)
+            flags=PaymentFlag.NONE, mask=PaymentFlag.MASK_HIDDEN | PaymentFlag.STATE_PREPARING)
```

Comment:
I don't think there's any issue here per se but I have noticed that immediately after creating a new legacy payment invoice, there is no corresponding unpaid row appearing in the request_list like there used to be...

This suggests that there is some issue with the subsequent updating of an invoices flags from the `PaymentFlag.STATE_PREPARING` state to subsequent states (e.g. `PaymentFlag.STATE_UNPAID` would be the next in sequence after successful tip filter registration)...

Reply:
![image](https://user-images.githubusercontent.com/8123976/226508989-069c4cbb-b937-479b-93c4-65f53c2ddfb7.png)